docs/configuration.rst

```diff
@@ -272,6 +272,8 @@ Let's say we want to convert all unicode typed properties to TextAreaFields inst
             type_map = ClassMap({sa.Unicode: TextAreaField})
 
 
+In case the type_map dictionary values are not inherited from WTForm field class, they are considered callable functions. They will called with the corresponding column as their only parameter.
```

Comment:
Small typo, "They will called" -> "These functions will be invoked" or "These functions will be called"